public/app/features/inspector/InspectErrorTab.tsx

```diff
@@ -18,5 +18,20 @@ export const InspectErrorTab: React.FC<InspectErrorTabProps> = ({ error }) => {
       </>
     );
   }
-  return <div>{error.message}</div>;
+  if (error.message) {
+    try {
+      const [msg, json] = error.message.split(/\{(.+)/);
```

Comment:
you can add the `{` to the capturing group so you don't need to add it later, right?

```suggestion
      const [msg, json] = error.message.split(/(\{.+)/);
```

Also, mostly styling but I would create a function that checks if the message contains a JSON, something like:

```js
if (error.message) {
  const { msg, json, hasJSON } = parseErrorMsg(error.message);
  return <div>{msg} {hasJSON ? <JSONFormatter ... /> : null}</div>
}
```

(where msg above is either what  you call `title` or `error.message`.